cicecore/cicedyn/infrastructure/ice_read_write.F90

```diff
@@ -1239,16 +1236,22 @@ subroutine ice_read_nc_xy(fid,  nrec,  varname, work,  diag, &
 !            status = nf90_inquire_dimension(fid,id,name=dimname,len=dimlen)
 !            write(nu_diag,*) subname,' Dim name = ',trim(dimname),', size = ',dimlen
 !         enddo
+         ! optional
+         missingvalue = 1.234e30
+         status = nf90_get_att(fid, varid, "_FillValue", missingvalue)
+!          call ice_check_nc(status, subname//' ERROR: Missing _FillValue', &
+!                            file=__FILE__, line=__LINE__)
```

Comment:
if (status != NC_ENOTATT) then
call ice_check_nc( ...

Reply:
See the version in my suggestions :)

Reply:
@apcraig  is the assignment of the missingvalue to 1.234e30 (which seems like an odd missingvalue) only to prevent uninitialized-variable errors under debug?

Reply:
This is an odd section of code that I'm not sure what to do with.  This is only used with some diag flags, so maybe infrequently.  I think we want to avoid missingvalue being undefined but there isn't a "good" value to set it to by default.  You probably want it set to something that will make the mask ineffective in the diagnostics.  There are lots of other ways to do this including checking the return code on the _FillValue then not masking the diagnostics if missingvalue is not defined.  But I feel like this is a reasonable workaround for now.  We can always fix it if it ever becomes a problem.

Reply:
That is fine w/ me. Thanks.

Reply:
I feel like we should be using spval or spval_dbl.